documentation/prerequisites.md

```diff
@@ -5,7 +5,7 @@ title: Prerequisites
 Following is the list of prerequisites for running Kubernetes with vSphere Cloud Provider:
 
 * We recommend Kubernetes version 1.6.5 and vSphere version 6.0.x
-* We support Photon, Ubuntu, CoreOs and Centos. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3
+* We support Photon, Ubuntu, CoreOs, Centos and RHEL OS. it has been tested on Photon 1.0 GA, Ubuntu 16.04, CoreOS 4.11.2 and Centos 7.3. 
```

Comment:
Let's remove "we support" from every line.
Just add bullet points.
* Minimum Kubernetes version
* Container Host operating systems -
** Photon - 1.0
** RHEL - version?
etc.